### DISCUSSION TITLE
Any plan to add login options?

### DISCUSSION THREAD

**Discussion by techguruagi** [FlowiseAI/Flowise#101] [Ideas]
At present, if i want to access api setup in a different server, i've to expose the entire workflow and any thirdparty can edit / delete it. (Also they have the access to all the keys added, which is a big security threat.). Why not provide an option and the dashboard is accessible only after the login ?

**Comment by HenryHengZJ** [comment]
Definitely, work in progress!

**Reply by techguruagi** [reply]
Appreciate it. Looking forward for the release. If you don't mind, possible to share the rough ETA ?

**Reply by HenryHengZJ** [reply]
PR opened as we speak: https://github.com/FlowiseAI/Flowise/pull/103 😄

**Reply by HenryHengZJ** [reply]
PR merged 🥳

**Reply by techguruagi** [reply]
Wow. Fantastic. Will test it. Thank you :-)
